killing-floor-2.kvp

```diff
@@ -22,7 +22,7 @@ App.ExecutableLinux=232130/Binaries/Win64/KFGameSteamServer.bin.x86_64
 App.WorkingDir=232130
 App.LinuxCommandLineArgs=
 App.WindowsCommandLineArgs=
-App.CommandLineArgs={{Map}}?Game={{Game}}?maxplayers={{$MaxUsers}} -Port={{$ApplicationPort1}} -QueryPort={{$ApplicationPort2}} -WebAdminPort={{$RemoteAdminPort}} -Multihome={{$ApplicationIPBinding}} {{$FormattedArgs}}
+App.CommandLineArgs={{Map}}?Game={{Game}}?Mutator="{{ServerMutators}}"?maxplayers={{$MaxUsers}} -Port={{$ApplicationPort1}} -QueryPort={{$ApplicationPort2}} -WebAdminPort={{$RemoteAdminPort}} -Multihome={{$ApplicationIPBinding}} {{$FormattedArgs}}
```

Comment:
I haven't tested myself yet, does this still load properly and keep the max players if you leave the Mutator blank?